confluent_kafka/src/Metadata.c

```diff
@@ -405,3 +419,215 @@ const char list_topics_doc[] = PyDoc_STR(
         " :param float timeout: Maximum response time before timing out, or -1 for infinite timeout.\n"
         " :rtype: ClusterMetadata \n"
         " :raises: KafkaException \n");
+
+
+static PyObject *
+c_group_members_to_py(Handle *self, const struct rd_kafka_group_member_info *c_members,
+                      int member_cnt)
+{
```

Comment:
style nit: open-brace should go on the function params line, like so:
`... int member_cnt) {`